<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GEditor</class>
 <widget class="QMainWindow" name="GEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1600</width>
     <height>800</height>
    </size>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNewLevel"/>
    <addaction name="actionNewProject"/>
    <addaction name="separator"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionCloseProject"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionAudio"/>
    <addaction name="actionSprite"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action50"/>
    <addaction name="action75"/>
    <addaction name="action100"/>
    <addaction name="action125"/>
    <addaction name="action150"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewProject"/>
   <addaction name="actionNewLevel"/>
   <addaction name="actionOpenProject"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionFind"/>
   <addaction name="separator"/>
   <addaction name="actionDefault"/>
   <addaction name="actionMove"/>
   <addaction name="separator"/>
   <addaction name="actionRotation"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewProject">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/newp.ico</normaloff>:/images/resources/images/icons/newp.ico</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionNewLevel">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/newl.ico</normaloff>:/images/resources/images/icons/newl.ico</iconset>
   </property>
   <property name="text">
    <string>New Level</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/open.ico</normaloff>:/images/resources/images/icons/open.ico</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/save.ico</normaloff>:/images/resources/images/icons/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/saveAs.ico</normaloff>:/images/resources/images/icons/saveAs.ico</iconset>
   </property>
   <property name="text">
    <string>Save As ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/closep.ico</normaloff>:/images/resources/images/icons/closep.ico</iconset>
   </property>
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/exit.ico</normaloff>:/images/resources/images/icons/exit.ico</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/import.ico</normaloff>:/images/resources/images/icons/import.ico</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/cut.ico</normaloff>:/images/resources/images/icons/cut.ico</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/copy.ico</normaloff>:/images/resources/images/icons/copy.ico</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/paste.ico</normaloff>:/images/resources/images/icons/paste.ico</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/options.ico</normaloff>:/images/resources/images/icons/options.ico</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/export.ico</normaloff>:/images/resources/images/icons/export.ico</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAudio">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/audio.ico</normaloff>:/images/resources/images/icons/audio.ico</iconset>
   </property>
   <property name="text">
    <string>Audio</string>
   </property>
  </action>
  <action name="actionSprite">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/sprites.ico</normaloff>:/images/resources/images/icons/sprites.ico</iconset>
   </property>
   <property name="text">
    <string>Sprite Creator</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/help.ico</normaloff>:/images/resources/images/icons/help.ico</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/about.ico</normaloff>:/images/resources/images/icons/about.ico</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/arrow.ico</normaloff>:/images/resources/images/icons/arrow.ico</iconset>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
   <property name="toolTip">
    <string>Default</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/drag.ico</normaloff>:/images/resources/images/icons/drag.ico</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="toolTip">
    <string>Move</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="action50">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>50%</string>
   </property>
  </action>
  <action name="action75">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>75%</string>
   </property>
  </action>
  <action name="action100">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </action>
  <action name="action150">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>150%</string>
   </property>
  </action>
  <action name="action125">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>125%</string>
   </property>
  </action>
  <action name="actionRotation">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/rotation.ico</normaloff>:/images/resources/images/icons/rotation.ico</iconset>
   </property>
   <property name="text">
    <string>Rotation</string>
   </property>
   <property name="toolTip">
    <string>Rotation</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/resources/images/icons/find.ico</normaloff>:/images/resources/images/icons/find.ico</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="toolTip">
    <string>Find</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>GEditor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>574</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
